src/uct/ib/mlx5/dc/dc_mlx5.c

```diff
@@ -1603,7 +1592,7 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     self->tx.ndci   = uct_dc_mlx5_iface_is_hw_dcs(self) ? 1 : config->ndci;
 
     if (config->dcis_initial_capacity == UCS_ULUNITS_AUTO) {
-        max_capacity = uct_dc_mlx5_iface_is_dci_rand(self) ? SIZE_MAX : 1;
+        max_capacity = uct_dc_mlx5_iface_is_dci_rand(self) ? SIZE_MAX : 0;
```

Comment:
needed since previously uct_dc_mlx5_iface_init_dcis_array() set array length to 0